graylog2-web-interface/src/util/FormsUtils.js

```diff
@@ -14,6 +14,11 @@ const FormUtils = {
         return input.value;
     }
   },
+
+  // Emulates a minimal input event that can be used with FormUtils#getValueFromInput()
+  inputEvent(name, value) {
```

Comment:
This looks quite nasty and as far as I see it is safe to remove. If we really need it, then we should use `CustomEvent` to get the same result but with an event object.

Reply:
I will remove it. :smile: